### PR TITLE
Update install and upgrade instruction using Tanzu CLI

### PR DESCRIPTION
**Pull request by dsharp-pivotal**
v1.0.0 of the Tanzu CLI has significant changes from prior versions. Update our instructions to reflect them.

Name the branches to merge this change with or enter "None".
main, 1.9

### CODE REVIEW

File: install-operator.html.md.erb [hunk 1]
```diff
@@ -453,11 +454,11 @@ To relocate images from the VMware Tanzu Network registry to a private registry:
    where:
     - `MY-REGISTRY` is your own image registry
     - `TARGET-REPOSITORY` is your target repository
-    - `TDS-VERSION` is the tag for the image bundle (e.g `1.9.0`). Each VMware MySQL Operator release corresponds to a TDS package version. For instance, MySQL Operator version 1.9.0 is part of TDS version 1.9.0. Insert the right version for your VMware MySQL Operator release.
+    - `TDS-VERSION` is the tag for the image bundle (e.g `1.11.0`). Each VMware MySQL Operator release corresponds to a TDS package version. For instance, MySQL Operator version 1.9.0 is part of TDS version 1.11.0. Insert the right version for your VMware MySQL Operator release.
```
Comment (RiaMahoney) [inline install-operator.html.md.erb:457]: Same as below comment
Did you intend to remove the second period from `e.g.`? 

Minor nitpick: can we change `right version` to `correct version`?

**Review by dyozie — APPROVED** [review]
Looks good after addressing Ria's comments.

File: install-operator.html.md.erb [hunk 2]
```diff
@@ -487,42 +488,65 @@ tanzu secret registry add <SECRET-NAME> \
...
+     --url <MY-REGISTRY>/<TARGET-REPOSITORY>/tds-packages:<TDS-VERSION> \
+     -n <NAMESPACE>
 ```
 
 where:
 - `PACKAGE-REPOSITORY-NAME` is a name chosen to identify this package repository
 - `MY-REGISTRY` is your own image registry
 - `TARGET-REPOSITORY` is your target repository
-- `TDS-VERSION` is the tag for the image bundle (e.g. `1.9.0`)
+- `TDS-VERSION` is the tag for the image bundle (e.g `1.11.0`). Each VMware MySQL Operator release corresponds to a TDS package version. For instance, MySQL Operator version 1.9.0 is part of TDS version 1.11.0. Insert the right version for your VMware MySQL Operator release.
```
[the remark this answers is not in the document]
Comment (dsharp-pivotal) [reply install-operator.html.md.erb]: Ah, the extended comment was copied from another location that was missing that period.

Fixed.

File: install-operator.html.md.erb (hunk 1 above)
Comment (dsharp-pivotal) [reply install-operator.html.md.erb]: Fixed.

**Pull request merged**